CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- Log level is now configurable using the `LOG_LEVEL` environment variable or `conjur.org/log-level` annotation.
```

Comment:
This repo follows the automated release process so shouldn't the unreleased section be empty with new changes going under v1.5.2? (or 1.6.0 if minor version bump is warranted)